### ISSUE TITLE
channel !

### ISSUE THREAD

**Issue by pipiche38** [fairecasoimeme/ZiGate#100]
Using ZGUI I put 
0x00000000 and do Set CMSK

Then I do Network State
got 0x8009 and Channel: 11

Then I do Start NWK
Got 0x8024 and Channel is 7

Redo Network State and git the same 0x8009 with Channel: 11

What are the difference between Channel from 0x8009 and channel from 0x8025 ?

```
Type: 0x8000 (Status)
  Length: 5
  Status: 0x00 (Success)
  SQN: 0x00
  Message: 
Type: 0x8000 (Status)
  Length: 5
  Status: 0x00 (Success)
  SQN: 0x00
  Message: 
Type: 0x8009 (Network State Response)
  Short Address: 0x0000
  Extended Address: 0x158D0001EDEDDE
  PAN ID: A307
  Ext PAN ID: 0x15A5A3C062B3D2F8
  Channel: 11
Type: 0x8024 (Network Up)
  Status: 0x04
  Short Address: 0x0000
  Extended Address: 0x158D0001EDEDDEA3
  Channel: 7
Type: 0x8000 (Status)
  Length: 5
  Status: 0x00 (Success)
  SQN: 0x00
  Message: 
```

**Comment by fairecasoimeme** [comment]
Hi,
exact, it's a mistake.
I'll fix that in the next firmware.
thanks

**Comment by pipiche38** [comment]
@fairecasoimeme Mistake where, in ZGUI, or in one of the Command 0x0024/0x8024 or 0x0009/0X8089 ?

**Comment by pipiche38** [comment]
I'm not sure I'm following you . As all the steps were done with ZGUI

(1) Set CMSK : 0x00000000 

So I can understand your point  of the 0x04 status aand so the channel is not good, but where the channel 7 came from ?

**Comment by pipiche38** [comment]
@fairecasoimeme could you kindly comment where this restrictions on the channels are coming from ?
If you set other channel than 11,15,19,20,25 ZiGate set default channel like 11.

For me the ZigBee channels supported by the NXP stack are 11 to 26, and in Europe we would benefit to use those ''11,15,16,21,22' as there overall with the Wifi are limited

**Comment by ISO-B** [comment]
I can't answer why channels are restricted, but restrictions are defined at ZigbeeNodeControlBridgeCoordinator.zpscfg

**Comment by pipiche38** [comment]
However, it works with Channel 26 in order to get the Livolo connected. 
So I don't feel the restriction really is that one

**Comment by ISO-B** [comment]
Channels available changed to version 3.0f. Currently available channels can be found:
https://github.com/fairecasoimeme/ZiGate/blob/76c328bb44a6fdd1774c86807b101c9c15edfebc/Module%20Radio/Firmware/src/ZiGate/Source/ZigbeeNodeControlBridge/ZigbeeNodeControlBridgeCoordinator.zpscfg#L405

**Comment by pipiche38** [comment]
And we agree that channel 0 is interpreted as all of those one .
